org.godotengine.Godot3.yaml

```diff
@@ -0,0 +1,133 @@
+app-id: org.godotengine.Godot3
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+  - org.freedesktop.Sdk.Extension.dotnet7
+command: godot
+
+build-options:
+  arch:
+    x86_64:
+      env:
+        # Only enable link-time optimization when targeting x86_64
+        # (causes issues on other architectures)
+        SCONS_FLAGS_EXTRA: lto=full
+
+  append-path: /usr/lib/sdk/dotnet7/bin
+
+  append-ld-library-path: /usr/lib/sdk/dotnet7/lib
+
+  env:
+    PKG_CONFIG_PATH: /app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig
+
+    # Will be appended to the version string displayed in the editor and command-line help
+    BUILD_NAME: flathub
+
+    DOTNET_CLI_TELEMETRY_OPTOUT: True
+
+    DOTNET_NOLOGO: True
+
+    DOTNET_ROOT: /usr/lib/sdk/dotnet7/lib
+
+    # SCons flags common to all builds
+    # Shouldn't be quoted when used as it's a single string, not an array
+    SCONS_FLAGS: >
+      platform=linuxbsd
+      CCFLAGS=-I/app/include
+      prefix=/app
+      unix_global_settings_path=/app
+      progress=no
+      builtin_freetype=no
+      builtin_graphite=no
+      builtin_harfbuzz=no
+      builtin_libogg=no
+      builtin_libpng=no
+      builtin_libtheora=no
+      builtin_libvorbis=no
+      builtin_libwebp=no
+      builtin_openssl=no
+      builtin_libvpx=no
+      builtin_zlib=no
+      udev=no
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --device=all
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
This needs an exception in the linter

Reply:
I just created a PR for the linter: flathub/flatpak-builder-lint#190